web/cobrands/fixmystreet-uk-councils/assets.js

```diff
@@ -197,6 +199,57 @@ fixmystreet.assets.brent.not_found = function(layer) {
     fixmystreet.message_controller.road_found(layer);
 };
 
+fixmystreet.assets.brent.no_asset_message_defaults = {
+    'Parks_and_Open_Spaces' : '#js-brent-park-services',
+    'Highways' : '#js-brent-road-services'
+};
+
+fixmystreet.assets.brent.parkRoadIsFound = {
+    park: false,
+    road: false,
+    isParkorRoad: function() { return this.park || this.road; }
+};
+
+fixmystreet.assets.brent.found_filter = function(layer) {
+    if (fixmystreet.reporting.selectedCategory().category === 'Grass verges / shrub beds - littering') {
+        if (layer.features[0] && layer.features[0].type === 'Parks_and_Open_Spaces') {
+            fixmystreet.assets.brent.parkRoadIsFound.park = true;
+        }
+        if (layer.features[0] && layer.features[0].type === 'Highways') {
+            fixmystreet.assets.brent.parkRoadIsFound.road = true;
+        }
+        if (fixmystreet.assets.brent.parkRoadIsFound.isParkorRoad()) {
```

Comment:
I don't think this `if` is needed, because one of the previous two ifs will have always been true (as this function is called by one of those two layers).

Reply:
After discussion on slack, just confirming I meant remove the `if` itself - so that the road_found is always called - not remove the whole if clause :)